tests/unit/output/test_local.py

```diff
@@ -1,25 +1,52 @@
-from mock import patch
+import os
 
+from mock import patch
 from dvc.output import OutputLOCAL
 from dvc.remote.local import RemoteLOCAL
 from dvc.stage import Stage
+from dvc.utils import relpath
 from tests.basic_env import TestDvc
 
 
-class TestOutputLOCAL(TestDvc):
+class OutputLOCALBase(TestDvc):
     def _get_cls(self):
         return OutputLOCAL
 
-    def _get_output(self):
-        stage = Stage(self.dvc)
-        return self._get_cls()(stage, "path", cache=False)
+    def _get_output(self, wdir=None, path="path"):
+        stage = Stage(self.dvc, wdir=wdir if wdir else os.curdir)
```

Comment:
The original issue this PR aims to solve was about outputs in external repo and thus outside it. Testing with stage of the same repo might not cover that or might break in the future.

Reply:
Agreed. We could have two tests. One with `Stage(dvc_repo)` to test that it works correctly when in the repo, and one more test with `Stage(erepo.dvc)` to test that it works when outside of the repo. Should be enough.